### ISSUE TITLE
Add a possibility to save information about devices, their sensors and data

### ISSUE THREAD

**Issue by GUSAR1T0** [MERA-IOT-COURSE/Group3_IoT_Services#13]
# Main description

Implement class that will handle data from devices and their sensors. 

## Requirements

Store data (messages) which should be sent and could be got.

### Common

1. Create `Messages` interface in `Common/src/utils/protocol/messages.js`:
* `constructor()` -> init `collection` variable (`messages`)
* `function initialize() : void` -> initialize collection if it doesn't exist 
* `function get(message : stirng) : array of dictionaries with message data` -> get all data by message name
* `function save(message : string, data : dictionary with message data) : void` -> save data
2. Add interface into export of `index.js` for Common module

### Device (lowdb)

1. Implement Common interface (`Device/src/backend/protocol/messages.js`), similar code of work with lowdb is for ConfigurationProfile class (`Device/src/backend/profile/config.js`)
2. Location of file with messages: messages.json
3. Example of format:

```
{
    "MESSAGE1": [
        {
            "id": 1,  // autoincrement(1,1)
            "data": 
            {
                "value": 10,
                "time": 20181216
            }
        }
    ],
    "MESSAGE2": [ ... ]
}
```

### Server (MongoDB/Mongoose)

1. Implement Common interface (`Server/src/backend/protocol/messages.js`), similar code of work with MongoDB is for ConfigurationProfile class (`Server/src/backend/profile/config.js`)
2. Collection: messages
3. Schemas should describe `id` (numeric, autoincrement(1,1)) and `data` (dictionary, complex content)

**Comment by GUSAR1T0** [comment]
The issue is postponed because it's blocked by #40 and #41 tickets.
